package.json

```diff
@@ -60,6 +60,7 @@
     "react-dom": "^16.0.0",
     "react-router": "^4.2.0",
     "react-router-dom": "^4.2.2",
+    "react-scrollbar": "^0.5.1",
```

Comment:
Remove unnecessary include

Reply:
npm install wiped it out on my local repo. I don't see it in the package.json anymore, and the change should come through with my next commit.

Reply:
Sounds good, it should wipe out the other references that came in then